numba/cloudpickle/cloudpickle.py

```diff
@@ -65,21 +65,8 @@
 except ImportError:
     _typing_extensions = Literal = Final = None
 
-if sys.version_info >= (3, 5, 3):
```

Comment:
This is a vendored cloudpickle, right? Maybe we should just retain their stuff verbatim rather than "forking" it? This will surely also make future cloudpickle updates easier? Maybe there is a version of cloudpickle already that has dropped 3.7 support?

Reply:
Yes indeed starting with cloudpickle v2.2.0 they changed this line for python 3.8+ see https://github.com/cloudpipe/cloudpickle/blob/f31859b1dd83fa691f4f7f797166b262c9acb8e7/cloudpickle/cloudpickle.py#L68

Looks like numba’s cloudpickle has had a bug fix and typo fixed since the censored version was last updated to v1.6.0 see https://github.com/numba/numba/commits/main/numba/cloudpickle/cloudpickle.py

Reply:
@jamesobutler I am not sure yet, I think @sklam is most experienced in the cloudpickle integration, so I would like to wait for their commentary before attempting to reach consensus.

Reply:
i prefer to handle cloudpickle separately.

Reply:
Yes, please revert the changes to cloudpickle and submit a separate PR. Thank you for your effort.

Reply:
Cloudpickle changes have now been removed from this PR.